tests/CoinAddressValidationTests.cpp

```diff
@@ -11,6 +11,12 @@
 
 namespace TW {
 
+TEST(Coin, CoinTypes) {
+    // Verify number of coins.  Needs to be updated if a new coin is added.
+    std::set<TWCoinType> coins = getCoinTypes();
+    EXPECT_EQ(coins.size(), 57);
```

Comment:
can we remove this test? we used to have similar test in assets repo, quite annoying to update this number each number